tools/website.sh

```diff
@@ -1,7 +1,3 @@
 #! /bin/bash
 
-NODE_VER=`node -v`
-
-if [[ $NODE_VER =~ ^v6 && $TRAVIS_PULL_REQUEST == "false" && $TRAVIS_BRANCH == "master" ]]; then
```

Comment:
this if would need to be restored. We should run this only for master.

Reply:
This will be run only on master look at the `condition` in the pipeline script.